health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/components/fetchFromMicroplan.js

```
@@ -0,0 +1,166 @@
+      }
+    };
+    const navigateTimeout = setTimeout(() => {
+      fetchTemplates();
+    }, 3000);
+    return () => clearTimeout(navigateTimeout); // Cleanup timeout
+
+  }, [data]);
+
+
+useEffect(async()=>{
+  if(templates&&templates?.completed){
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)